### ISSUE TITLE
Duplicate "Revert" links on Revisions table in CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10160]
## Description
On the Revision screens in the CMS each row in the table has a call to action that says "Revert". This can cause duplicate links on the same page when there are multiple previous revisions.  Ideally each link should have a distinct, unique text to indicate to the user where the link would direct them - "Revert to version XXX"

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/184167308-fccf099b-563d-47d0-aa60-768a6bced61b.png)

## Accessibility Standard

WCAG version 2.0 AAA, [Criterion 2.4.9](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)

## Acceptance Criteria
- [ ] UX/IA review to determine content updates for links
- [ ] Technical review
- [ ] Change management consulted
- [ ] Implementation ticket created

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`
